hbp_validation_framework/utils.py

```diff
@@ -20,7 +21,8 @@
 from datetime import datetime
 from . import TestLibrary, ModelCatalog
 from .datastores import CollabDataStore
-
+from fpdf import FPDF
```

Comment:
I suggest making these dependencies optional, i.e. `generate_report()` produces an error message saying "Please install these packages ...." if they are not installed.

Reply:
Will be done in https://github.com/apdavison/hbp-validation-client/issues/18